brainscore/benchmarks/baker2022.py

```diff
@@ -0,0 +1,131 @@
+        self._visual_degrees = 8.8
+        self._number_of_trials = 1
+
+        super(_Baker2022AccuracyDelta, self).__init__(
+            identifier=f'Baker2022{dataset}-accuracy_delta', version=1,
+            ceiling_func=lambda: self._ceiling(assembly=self._assembly),
+            parent='Baker2022',
+            bibtex=BIBTEX)
+
+    def __call__(self, candidate: BrainModel):
+        choice_labels = set(self._assembly['truth'].values)
+        choice_labels = list(sorted(choice_labels))
+        candidate.start_task(BrainModel.Task.label, choice_labels)
+        stimulus_set = place_on_screen(self._assembly.stimulus_set, target_visual_degrees=candidate.visual_degrees(),
+                                       source_visual_degrees=self._visual_degrees)
+        if self.orientation == "inverted":
+            inverted_stimuli = stimulus_set[stimulus_set["orientation"] == "inverted"]
+            labels = candidate.look_at(inverted_stimuli, number_of_trials=self._number_of_trials)
+            inverted_assembly = self._assembly[self._assembly["orientation"] == "inverted"]
+            raw_score = self._metric(labels, inverted_assembly)
+            ceiling = self._ceiling(inverted_assembly)
+        else:
+            labels = candidate.look_at(stimulus_set, number_of_trials=self._number_of_trials)
+            raw_score = self._metric(labels, self._assembly)
+            ceiling = self._ceiling(self._assembly)
+        score = raw_score / ceiling
+
+        # cap score at 1 if ceiled score > 1
+        if score[(score['aggregation'] == 'center')] > 1:
+            score.__setitem__({'aggregation': score['aggregation'] == 'center'}, 1)
+
+        score.attrs['raw'] = raw_score
+        score.attrs['ceiling'] = ceiling
+        return score
+
+
+def Baker2022AccuracyDeltaFrankenstein():
+    return _Baker2022AccuracyDelta(dataset='normal', image_types=["w", "f"])
+
+
+def Baker2022AccuracyDeltaFragmented():
+    return _Baker2022AccuracyDelta(dataset='normal', image_types=["w", "o"])
+
+
+def Baker2022InvertedAccuracyDelta():
+    return _Baker2022AccuracyDelta(dataset='inverted', image_types=["w", "f"])
+
+
+def load_assembly(dataset):
+    assembly = brainscore.get_assembly(f'Baker2022_{dataset}_distortion')
+    return assembly
+
+
+# ceiling method:
+class SplitHalvesConsistencyBaker:
+    def __init__(self, num_splits: int, split_coordinate: str, consistency_metric, image_types):
+        """
+        :param num_splits: how many times to create two halves
+        :param split_coordinate: over which coordinate to split the assembly into halves
+        :param consistency_metric: which metric to use to compute the consistency of two halves
+        """
+        self.num_splits = num_splits
+        self.split_coordinate = split_coordinate
+        self.consistency_metric = consistency_metric
+        self.image_types = image_types
+
+    def __call__(self, assembly) -> Score:
```

Comment:
```suggestion
    def __call__(self, assembly: DataAssembly) -> Score:
```
might need to import from brainio